ui/components/app/tab-bar/tab-bar.js

```diff
@@ -16,10 +17,8 @@ const TabBar = (props) => {
           onClick={() => onSelect(key)}
         >
           <div className="tab-bar__tab__content">
-            <div className="tab-bar__tab__content__title">{content}</div>
-            <div className="tab-bar__tab__content__description">
-              {description}
-            </div>
+            <div className="tab-bar__tab__content__icon">{icon}</div>
+            <Typography>{content}</Typography>
```

Comment:
I think you'll still need to supply `TYPOGRAPHY.H4` to the variant prop to get the size the design is looking for

Reply:
@brad-decker done.